src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,49 +2,58 @@ import React from 'react';
 import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
-type Film = {
+type Props = {
   movie: Movie;
 };
 
-export const MovieCard: React.FC<Film> = ({ movie }) => (
-  <div className="card" data-cy="Movie">
-    <div className="card-image">
-      <figure className="image is-4by3">
-        <img
-          data-cy="MovieImage"
-          src={`${movie.imgUrl}`}
-          alt="Film logo"
-        />
-      </figure>
-    </div>
+export const MovieCard: React.FC<Props> = ({ movie }) => {
+  const {
+    title,
+    imgUrl,
+    description,
+    imdbUrl,
+  } = movie;
 
-    <div className="card-content">
-      <div className="media">
-        <div className="media-left">
-          <figure className="image is-48x48">
-            <img src="images/imdb-logo.jpeg" alt="imdb" />
-          </figure>
-        </div>
+  return (
+    <div className="card" data-cy="Movie">
+      <div className="card-image">
+        <figure className="image is-4by3">
+          <img
+            data-cy="MovieImage"
+            src={`${imgUrl}`}
```

Comment:
It works without interpolation as well, just the variable => src={imgUrl}

Reply:
I see, thanks!